pontoon/base/models.py

```diff
@@ -1,11 +1,15 @@
+from __future__ import unicode_literals
 from __future__ import division
+from django.utils.encoding import python_2_unicode_compatible
 
 import hashlib
 import logging
 import math
 import os.path
 import re
-import urllib
+from six.moves.urllib.parse import urlencode
+from six.moves import reduce
+from six.moves.urllib.parse import urlparse
```

Comment:
Please rewrite this to:

```python
from six.moves import reduce
from six.moves.urllib.parse import (urlencode, urlparse)
```

and move it to after ``from dirtyfields import DirtyFieldsMixin`` (as ``six`` is a dependency and not from the standard library).